src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -322,11 +322,13 @@ function WorkspaceMembersPage(props) {
                         <Text style={styles.peopleBadgeText}>{props.translate('common.admin')}</Text>
                     </View>
                 ) : null,
-                avatar: {
-                    source: UserUtils.getAvatar(details.avatar, accountID),
-                    name: details.login,
-                    type: CONST.ICON_TYPE_AVATAR,
-                },
+                icons: [
+                    {
+                        source: UserUtils.getAvatar(details.avatar, accountID),
+                        name: details.login,
+                        type: CONST.ICON_TYPE_AVATAR,
+                    },
+                ],
```

Comment:
We forgot to pass the id of account as separate field in object https://github.com/Expensify/App/issues/29900#issuecomment-1769124767